src/Administration/Resources/app/administration/src/module/sw-flow/page/sw-flow-detail/index.js

```diff
@@ -305,9 +305,11 @@ export default {
             return State.commit('swFlowState/setFlow', flow);
         },
 
-        getDetailFlow() {
+        async getDetailFlow() {
             this.isLoading = true;
-            Shopware.State.dispatch('swFlowState/fetchTriggerActions');
+
+            const triggerActionsPromise = Shopware.State.dispatch('swFlowState/fetchTriggerActions');
+            await triggerActionsPromise;
```

Comment:
@ennasus4sun What's up with this variable, couldn't his be awaited directly?
```suggestion
            await Shopware.State.dispatch('swFlowState/fetchTriggerActions');
```

Reply:
yes 👍🏻